jobs/v4/src/main/java/com/example/jobs/JobSearchCommuteSearch.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jobs;
+
+// [START job_search_commute_search]
+
+import com.google.cloud.talent.v4beta1.CommuteFilter;
+import com.google.cloud.talent.v4beta1.CommuteMethod;
+import com.google.cloud.talent.v4beta1.Job;
+import com.google.cloud.talent.v4beta1.JobQuery;
+import com.google.cloud.talent.v4beta1.JobServiceClient;
+import com.google.cloud.talent.v4beta1.RequestMetadata;
+import com.google.cloud.talent.v4beta1.SearchJobsRequest;
+import com.google.cloud.talent.v4beta1.SearchJobsResponse;
+import com.google.cloud.talent.v4beta1.TenantName;
+import com.google.cloud.talent.v4beta1.TenantOrProjectName;
+import com.google.protobuf.Duration;
+import com.google.type.LatLng;
+
+import java.io.IOException;
+
+public class JobSearchCommuteSearch {
```

Comment:
nit - class name seems a bit wordy.  JobByCommuteSearch perhaps.  (not required)

Reply:
Only class or all other search classes in general? I can remove the prefix [JobSearch] from Search classes